roles/openshift_master_facts/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+openshift_master_default_subdomain: "{{ lookup('oo_option', 'openshift_master_default_subdomain') | default(None, true) }}"
```

Comment:
Do we need to pull only this change into release-1.3? I think the other changes in this PR aren't relevant to release-1.3.